twoliter/embedded/Makefile.toml

```diff
@@ -857,6 +862,9 @@ export BUILDSYS_VARIANT_PLATFORM="${BUILDSYS_VARIANT_PLATFORM:?}"
 export BUILDSYS_VARIANT_RUNTIME="${BUILDSYS_VARIANT_RUNTIME:?}"
 export BUILDSYS_VARIANT_FAMILY="${BUILDSYS_VARIANT_FAMILY:?}"
 export BUILDSYS_VARIANT_FLAVOR="${BUILDSYS_VARIANT_FLAVOR}"
+export BUILDSYS_VERSION_BUILD="${BUILDSYS_VERSION_BUILD}"
+export BUILDSYS_VERSION_BUILD_TIMESTAMP="${BUILDSYS_VERSION_BUILD_TIMESTAMP}"
+export BUILDSYS_VERSION_BUILD_EPOCH="${BUILDSYS_VERSION_BUILD_EPOCH}"
```

Comment:
Aren't these also needed in `build-kit`?

Reply:
See above comment, we won't need these in `build-kit` as they're already included in `build-package`